### DISCUSSION TITLE
WRONG color set, wtf?

### DISCUSSION THREAD

**Discussion by pragavame** [google/model-viewer#3902] [Q&A]
![wtf color](https://user-images.githubusercontent.com/99244643/197382567-63223582-8d75-494b-94f3-5ae1f7d3191b.png)

You can see which color is selected in the color palette, and another is set on the model.

**Comment by elalish** [comment]
Tthe glTF baseColorFactor is linear, not sRGB like the texture, which is probably the biggest reason for the difference. Our color picker is probably not representing that correctly. 

If you want your base color to show up unaltered in the render, you'll need to use the `kHR_materials_unlit` extension. With PBR, the output color is affected by the applied lighting and tone mapping in order to look realistic. Or you may also want to try changing exposure.

**Reply by elalish** [reply]
I fixed the missing sRGB conversion in the color pickers: https://github.com/google/model-viewer/pull/3905

**Reply by elalish** [reply]
Of course the colors still won't match exactly due to lighting:
<img width="920" alt="image" src="https://user-images.githubusercontent.com/1649964/197618763-02a233fb-5e51-4837-b3b3-97a838f01b28.png">
